### PR TITLE
Search bar is created in header

### PR DESCRIPTION
**Pull request by federicomontillo**
Be careful with changes in layouts!

### CODE REVIEW

File: messages/en.json [hunk 1]
```diff
@@ -18,13 +18,20 @@
     "subtitle2": "This is a text"
   },
   "PopUpNewsletter": {
-    "text": "Subscribe to the newsletter and find out about the latest news and receive important discount coupons."
+    "text": "Subscribe to the newsletter and find out about the latest news and receive important discount coupons.",
+    "Buscando": "Buscando..."
```
Comment (ccarapezza) [inline messages/en.json:22]: Quedó invertida esta traducción

File: messages/es.json [hunk 2]
```diff
@@ -18,13 +18,20 @@
     "subtitle2": "Esto es un texto"
   },
   "PopUpNewsletter": {
-    "text": "Suscríbete al newsletter y entérate de las últimas novedades y recibí importantes cupones de descuento."
+    "text": "Suscríbete al newsletter y entérate de las últimas novedades y recibí importantes cupones de descuento.",
+    "Buscando": "Searching..."
```
Comment (ccarapezza) [inline messages/es.json:22]: Quedó invertida esta traducción

File: src/components/store/navbar/Navbar.tsx [hunk 3]
```diff
@@ -8,6 +8,7 @@ import { Box } from '@mui/material';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faBars, faTimes } from '@fortawesome/free-solid-svg-icons';
 import LanguageSwitcher from '@/components/main-ui/LanguageSwitcher';
+import SearchBar from '../search-bar/SearchBar';
```
Comment (ccarapezza) [inline src/components/store/navbar/Navbar.tsx:11]: Preferible usar @/components/store/search-bar/SearchBar

**Review by ccarapezza — CHANGES_REQUESTED** [review]
Algunos cambios menores requeridos antes de mergear

File: messages/es.json (hunk 2 above)
Comment (federicomontillo) [reply messages/es.json]: Ready, the text was duplicated.

File: messages/en.json (hunk 1 above)
Comment (federicomontillo) [reply messages/en.json]: Ready, the text was duplicated.

File: src/components/store/navbar/Navbar.tsx (hunk 3 above)
Comment (federicomontillo) [reply src/components/store/navbar/Navbar.tsx]: Ready!

**Review by ccarapezza — APPROVED** [review]
Ok!

**Pull request merged**